src/utils/prisma-helpers.ts

```diff
@@ -0,0 +1,55 @@
+import prisma from '@/lib/prisma';
+
+/**
+ * Get a landlord by email address
+ *
+ * @param email - The email address of the landlord
+ * @returns A promise that resolves with the landlord
+ */
+export const getLandlordByEmail = async (email: string) => {
+  try {
+    return await prisma.landlord.findUnique({
+      where: {
+        email,
+      },
+    });
+  } catch {
+    return null;
+  }
+};
+
+/**
+ * Get a password reset token by email address
+ *
+ * @param email - The email address of the landlord
+ * @returns A promise that resolves with the password reset token
+ */
+export const getPasswordResetTokenByEmail = async (email: string) => {
+  try {
+    return await prisma.passwordResetToken.findFirst({
+      where: {
+        email,
+      },
+    });
+  } catch {
+    return null;
+  }
+};
+
+/**
+ * Get the associated password reset token data for a given token
+ *
+ * @param token - The token to search for
+ * @returns A promise that resolves with the password reset token data
+ */
+export const getPasswordResetDataByToken = async (token: string) => {
```

Comment:
IMO, how about renaming getPasswordResetDataByToken to getPasswordResetTokenByToken for consistency?
(like: getPasswordResetTokenByEmail)

Reply:
Thank you for pointing it out. I kept including the word, data, but I renamed it to getPasswordResetTokenDataByToken for more clarity.